### ISSUE TITLE
Add or Hide Field in Form Client

### ISSUE THREAD

**Issue by rcorzogutierrez** [hql287/Manta#178]
- [ ] I'm requesting a new feature.

- [ ] Windows

I'm using Windows 10 Fall Creator

In my case, the invoices that I generate are for clients in person, not for business. Sometimes I need more fields than the client form allows me, such as Address, Work Phone, Referred by etc. These options could be added in the invoice setting

**Comment by hql287** [comment]
Hey Rafael @rcorzogutierrez, sorry for taking a bit long to get back to you. Was a little busy shipping v1.1.2. 
This shouldn't be hard to do. Would you like to work on a PR for this? :)

**Comment by rcorzogutierrez** [comment]
Hi @hql287 I'm sorry to disappoint you, but I do not have the necessary knowledge for this. My skills in electron are very basic. React is my next challenge, for now Angular5 and Firebase is what I can offer.

**Comment by rcorzogutierrez** [comment]
excellent idea @hql287 , it would be very useful. I am very motivated with this project and I hope in the near future to make significant contributions

**Comment by hql287** [comment]
>  I am very motivated with this project and I hope in the near future to make significant contributions.

@rcorzogutierrez Yeah, I'm always looking forward to learning from other developers so that would be great! 👍

**Comment by hql287** [comment]
Ok, this should go into the next release
